packages/terra-list/docs/SingleSelectSections.md

```diff
@@ -0,0 +1,31 @@
+# Single Select List
```

Comment:
* Single Select List with sections

Same comment as above for description. Should also note that the single selection is across all sections and not a single select within each section.